exotic/output_files.py

```diff
@@ -46,8 +46,8 @@ def final_planetary_params(self, phot_opt, vsp_params, comp_star=None, comp_coor
                                                          f"{round_to_2(self.fit.errors['rprs'])}",
             "Transit depth (Rp/Rs)^2": f"{round_to_2(100. * (self.fit.parameters['rprs'] ** 2.))} +/- "
                                        f"{round_to_2(100. * 2. * self.fit.parameters['rprs'] * self.fit.errors['rprs'])} [%]",
-            "Semi Major Axis/Star Radius (a/Rs)": f"{round_to_2(self.fit.parameters['ars'], self.fit.errors['ars'])} +/- "
-                                                  f"{round_to_2(self.fit.errors['ars'])} ",
+            "Orbital Inclination (inc)": f"{round_to_2(self.fit.parameters['inc'], self.fit.errors['inc'])} +/- "
```

Comment:
This whole thing looks like maybe a good use case for Jinja, but I'm not sure we want to add a new dependency...

Reply:
Example: https://www.packetswitch.co.uk/network-device-templating-using-jinja-and-python/